### ISSUE TITLE
error cloning environment

### ISSUE THREAD

**Issue by bgatessucks** [conda/conda#2351]
`conda create --name post001 --clone 1.8.2` gives:

``` text
Traceback (most recent call last):
  File "C:\dev\bin\Anaconda\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\cli\main.py", line 139, in m
ain
    args_func(args, p)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\cli\main.py", line 146, in a
rgs_func
    args.func(args, p)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\cli\main_create.py", line 49
, in execute
    install.install(args, parser, 'create')
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\cli\install.py", line 247, i
n install
    clone(args.clone, prefix, json=args.json, quiet=args.quiet, index=index)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\cli\install.py", line 84, in
 clone
    quiet=quiet, index=index)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\misc.py", line 246, in clone
_env
    sorted_dists = r.dependency_sort(dists)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\resolve.py", line 729, in de
pendency_sort
    depends = lookup(value)
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\resolve.py", line 724, in lo
okup
    return set(ms.name for ms in self.ms_depends(value + '.tar.bz2'))
  File "C:\dev\bin\Anaconda\lib\site-packages\conda\resolve.py", line 573, in ms
_depends
    deps = [MatchSpec(d) for d in self.index[fn].get('depends', [])]
KeyError: 'pymkl-0.1.0.post051.gff66387-py27_0.tar.bz2'
```

Info:

``` text

             platform : win-64
        conda version : 4.0.5
  conda-build version : 1.14.1
       python version : 2.7.11.final.0
     requests version : 2.9.1
     root environment : C:\dev\bin\Anaconda  (writable)
  default environment : C:\dev\bin\Anaconda\envs\1.8.2
     envs directories : C:\dev\bin\Anaconda\envs
        package cache : C:\dev\bin\Anaconda\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False
```

**Comment by kalefranz** [comment]
@bgatessucks Can you provide the output of `conda list -n 1.8.2`?

**Comment by bgatessucks** [comment]
`conda list -n 1.8.2` gives:

```text
# packages in environment at C:\dev\bin\Anaconda\envs\1.8.1.post012:
#
alabaster                 0.7.3                    py27_0    <unknown>
babel                     1.3                      py27_0    <unknown>
backports.ssl-match-hostname 3.4.0.2                   <pip>
beautiful-soup            4.3.2                    py27_0    http://conda:82/ci/win-64/beautiful-soup-4.3.2-py27_0.tar.bz2
beautifulsoup4            4.3.2                     <pip>
bottleneck                1.0.0                np19py27_0    <unknown>
colorama                  0.3.3                    py27_0    <unknown>
configobj                 5.0.6                    py27_0    <unknown>
coverage                  4.0                      py27_0    defaults
cvxopt                    1.1.7                np19py27_1    http://conda:82/ci/win-64/cvxopt-1.1.7-np19py27_1.tar.bz2
cvxpy                     0.2.22               np19py27_0    http://conda:82/ci/win-64/cvxpy-0.2.22-np19py27_0.tar.bz2
cycler                    0.9.0                    py27_0    defaults
cython                    0.22                     py27_0    http://conda:82/ci/win-64/cython-0.22-py27_0.tar.bz2
cython_gsl                0.2.1.post29.gc2ffb843          py27_2    http://conda:82/ci/win-64/cython_gsl-0.2.1.post29.gc2ffb843-py27_2.tar.bz2
cythongsl                 0.2.1                     <pip>
cytoolz                   0.7.2                    py27_0    http://conda:82/ci/win-64/cytoolz-0.7.2-py27_0.tar.bz2
dateutil                  2.4.1                    py27_0    defaults
decorator                 4.0.4                    py27_0    defaults
derapproximator           0.52                      <pip>
dill                      0.2.3                    py27_0    http://conda:82/ci/win-64/dill-0.2.3-py27_0.tar.bz2
distarray                 v0.5.0.post23.g9ed2fba          py27_0    http://conda:82/ci/win-64/distarray-v0.5.0.post23.g9ed2fba-py27_0.tar.bz2
docutils                  0.12                     py27_0    http://conda:82/ci/win-64/docutils-0.12-py27_0.tar.bz2
ecos                      1.0.4.post96.g12d38fa      np19py27_1    http://conda:82/ci/win-64/ecos-1.0.4.post96.g12d38fa-np19py27_1.tar.bz2
faulthandler              2.4                      py27_0    http://conda:82/ci/win-64/faulthandler-2.4-py27_0.tar.bz2
freezegun                 0.2.2                    py27_1    http://conda:82/ci/win-64/freezegun-0.2.2-py27_1.tar.bz2
funcdesigner              0.5201                    <pip>
funcsigs                  0.4                      py27_0    <unknown>
futures                   3.0.3                    py27_0    http://conda:82/ci/win-64/futures-3.0.3-py27_0.tar.bz2
hdf5                      1.8.15.1                      2    <unknown>
html5lib                  0.999                    py27_0    http://conda:82/ci/win-64/html5lib-0.999-py27_0.tar.bz2
ipython                   3.2.1                    py27_0    http://conda:82/ci/win-64/ipython-3.2.1-py27_0.tar.bz2
jinja2                    2.7.3.post98.g85820fc          py27_0    http://conda:82/ci/win-64/jinja2-2.7.3.post98.g85820fc-py27_0.tar.bz2
jpeg                      8d                        vc9_0  [vc9]  defaults
jsonschema                2.4.0                    py27_0    <unknown>
libpng                    1.6.17                    vc9_1  [vc9]  defaults
libpython                 1.0                      py27_1    http://conda:82/ci/win-64/libpython-1.0-py27_1.tar.bz2
libtiff                   4.0.6                     vc9_1  [vc9]  defaults
line-profiler             1.0                       <pip>
line_profiler             1.0.post5.ga2f484a          py27_0    http://conda:82/ci/win-64/line_profiler-1.0.post5.ga2f484a-py27_0.tar.bz2
lxml                      3.4.4                    py27_0    <unknown>
markupsafe                0.23                     py27_0    <unknown>
matplotlib                1.4.2                np19py27_0    defaults
mechanize                 0.2.5                    py27_2    http://conda:82/ci/win-64/mechanize-0.2.5-py27_2.tar.bz2
memory-profiler           0.33                      <pip>
memory_profiler           0.33                     py27_0    http://conda:82/ci/win-64/memory_profiler-0.33-py27_0.tar.bz2
mistune                   0.7                      py27_0    http://conda:82/ci/win-64/mistune-0.7-py27_0.tar.bz2
mkl                       0.1.0.post51.gff66387           <pip>
mock                      1.2.0                    py27_0    http://conda:82/ci/win-64/mock-1.2.0-py27_0.tar.bz2
mpi4py                    1.3.1.post152.g2b55fb2          py27_0    http://conda:82/ci/win-64/mpi4py-1.3.1.post152.g2b55fb2-py27_0.tar.bz2
mpltools                  0.1.post21.g1d4c339          py27_1    http://conda:82/ci/win-64/mpltools-0.1.post21.g1d4c339-py27_1.tar.bz2
mpmath                    0.19.post3.g5cb118a      np19py27_1    http://conda:82/ci/win-64/mpmath-0.19.post3.g5cb118a-np19py27_1.tar.bz2
msvc_runtime              1.0.1                     vc9_0  [vc9]  defaults
multipledispatch          0.4.8                    py27_0    http://conda:82/ci/win-64/multipledispatch-0.4.8-py27_0.tar.bz2
mxbase                    3.2.7                    py27_1    http://conda:82/ci/win-64/mxbase-3.2.7-py27_1.tar.bz2
networkx                  1.9.1                    py27_0    <unknown>
nlopt                     2.2.4                np19py27_1    http://conda:82/ci/win-64/nlopt-2.2.4-np19py27_1.tar.bz2
nose                      1.3.7                    py27_0    <unknown>
numdifftools              0.6.0                np19py27_1    http://conda:82/ci/win-64/numdifftools-0.6.0-np19py27_1.tar.bz2
numexpr                   2.4.3                np19py27_0    <unknown>
numpy                     1.9.2                    py27_2    defaults
numpydoc                  0.5.post1.g7c3a1dc          py27_0    http://conda:82/ci/win-64/numpydoc-0.5.post1.g7c3a1dc-py27_0.tar.bz2
openopt                   0.5.2                np19py27_4    http://conda:82/ci/win-64/openopt-0.5.2-np19py27_4.tar.bz2
openssl                   1.0.2d                    vc9_0  [vc9]  defaults
packaging                 15.3                     py27_0    http://conda:82/ci/win-64/packaging-15.3-py27_0.tar.bz2
pandas                    0.15.2               np19py27_1    http://conda:82/ci/win-64/pandas-0.15.2-np19py27_1.tar.bz2
patsy                     0.3.0                np19py27_0    <unknown>
pbr                       1.3.0                    py27_0    http://conda:82/ci/win-64/pbr-1.3.0-py27_0.tar.bz2
pip                       7.1.2                    py27_0    defaults
protobuf                  3.0.0a3.post418+g0cb84ee          py27_0    http://conda:82/ci/win-64/protobuf-3.0.0a3.post418+g0cb84ee-py27_0.tar.bz2
psutil                    2.2.1                    py27_0    <unknown>
pyamg                     2.2.1                np19py27_0    http://conda:82/ci/win-64/pyamg-2.2.1-np19py27_0.tar.bz2
pydevd                    4.0.5                    py27_0    http://conda:82/ci/win-64/pydevd-4.0.5-py27_0.tar.bz2
pygments                  2.0.2                    py27_0    <unknown>
pymkl                     0.1.0.post051.gff66387          py27_0    http://conda:82/ci/win-64/pymkl-0.1.0.post051.gff66387-py27_0.tar.bz2
pyodbc                    3.0.10                   py27_0    http://conda:82/ci/win-64/pyodbc-3.0.10-py27_0.tar.bz2
pyparsing                 2.0.1                    py27_0    defaults
pyqt                      4.11.4                   py27_4    defaults
pyreadline                2.0                      py27_0    <unknown>
pytables                  3.2.0                np19py27_0    <unknown>
python                    2.7.11                        0    defaults
python-dateutil           2.4.2                    py27_0    <unknown>
python-ntlm               1.1.0                    py27_0    http://conda:82/ci/win-64/python-ntlm-1.1.0-py27_0.tar.bz2
pythondotnet              2.0.0.2.post1.b46375e2          py27_0    http://conda:82/ci/win-64/pythondotnet-2.0.0.2.post1.b46375e2-py27_0.tar.bz2
pytz                      2015.7                   py27_0    defaults
pywin32                   219                      py27_0    <unknown>
pyzmq                     14.7.0                   py27_0    <unknown>
qt                        4.8.7                     vc9_5  [vc9]  defaults
requests                  2.7.0                    py27_0    <unknown>
requests-ntlm             0.0.2.2.post25.ge76408a          py27_1    http://conda:82/ci/win-64/requests-ntlm-0.0.2.2.post25.ge76408a-py27_1.tar.bz2
sarge                     0.1.3                    py27_1    http://conda:82/ci/win-64/sarge-0.1.3-py27_1.tar.bz2
scikit-learn              0.16.1               np19py27_0    <unknown>
scipy                     0.16.0              np110py27_0    defaults
scs                       1.0.2.post93.g8c3f9d7      np19py27_0    http://conda:82/ci/win-64/scs-1.0.2.post93.g8c3f9d7-np19py27_0.tar.bz2
seaborn                   0.6.0               np110py27_0    defaults
setuptools                18.5                     py27_0    defaults
sip                       4.16.9                   py27_2    defaults
six                       1.10.0                   py27_0    defaults
snowballstemmer           1.2.0                    py27_0    <unknown>
sortedcontainers          0.9.4                    py27_0    http://conda:82/ci/win-64/sortedcontainers-0.9.4-py27_0.tar.bz2
spacefuncs                0.52                      <pip>
sphinx                    1.3.1                    py27_0    <unknown>
sphinx-rtd-theme          0.1.7                     <pip>
sphinx_rtd_theme          0.1.7                    py27_0    <unknown>
sqlalchemy                1.0.9                    py27_0    defaults
ssl_match_hostname        3.4.0.2                  py27_0    <unknown>
statsmodels               0.6.1                np19py27_0    <unknown>
sympy                     0.7.6                    py27_0    <unknown>
tables                    3.2.0                     <pip>
tk                        8.5.18                    vc9_0  [vc9]  defaults
toolz                     0.7.2                    py27_0    <unknown>
tornado                   4.1                      py27_0    http://conda:82/ci/win-64/tornado-4.1-py27_0.tar.bz2
tqdm                      0.1.pre14.gda5fdbf7c0          py27_0    http://conda:82/ci/win-64/tqdm-0.1.pre14.gda5fdbf7c0-py27_0.tar.bz2
wheel                     0.26.0                   py27_1    defaults
xlrd                      0.9.4                    py27_0    http://conda:82/ci/win-64/xlrd-0.9.4-py27_0.tar.bz2
xlsxwriter                0.7.3                    py27_0    <unknown>
zlib                      1.2.8                     vc9_2  [vc9]  defaults
# packages in environment at C:\dev\bin\Anaconda\envs\1.8.2:
#
alabaster                 0.7.3                    py27_0    <unknown>
babel                     1.3                      py27_0    <unknown>
backports.ssl-match-hostname 3.4.0.2                   <pip>
beautiful-soup            4.3.2                    py27_0    http://conda:82/ci/win-64/beautiful-soup-4.3.2-py27_0.tar.bz2
beautifulsoup4            4.3.2                     <pip>
bottleneck                1.0.0                np19py27_0    <unknown>
colorama                  0.3.3                    py27_0    <unknown>
configobj                 5.0.6                    py27_0    <unknown>
coverage                  3.7.1                    py27_0    http://conda:82/ci/win-64/coverage-3.7.1-py27_0.tar.bz2
cvxopt                    1.1.7                np19py27_1    http://conda:82/ci/win-64/cvxopt-1.1.7-np19py27_1.tar.bz2
cvxpy                     0.2.22               np19py27_0    http://conda:82/ci/win-64/cvxpy-0.2.22-np19py27_0.tar.bz2
cython                    0.22                     py27_0    http://conda:82/ci/win-64/cython-0.22-py27_0.tar.bz2
cython_gsl                0.2.1.post29.gc2ffb843          py27_2    http://conda:82/ci/win-64/cython_gsl-0.2.1.post29.gc2ffb843-py27_2.tar.bz2
cythongsl                 0.2.1                     <pip>
cytoolz                   0.7.2                    py27_0    http://conda:82/ci/win-64/cytoolz-0.7.2-py27_0.tar.bz2
decorator                 4.0.6                    py27_0    defaults
derapproximator           0.52                      <pip>
dill                      0.2.3                    py27_0    http://conda:82/ci/win-64/dill-0.2.3-py27_0.tar.bz2
distarray                 v0.5.0.post23.g9ed2fba          py27_0    http://conda:82/ci/win-64/distarray-v0.5.0.post23.g9ed2fba-py27_0.tar.bz2
docutils                  0.12                     py27_0    http://conda:82/ci/win-64/docutils-0.12-py27_0.tar.bz2
ecos                      1.0.4.post96.g12d38fa      np19py27_1    http://conda:82/ci/win-64/ecos-1.0.4.post96.g12d38fa-np19py27_1.tar.bz2
faulthandler              2.4                      py27_0    http://conda:82/ci/win-64/faulthandler-2.4-py27_0.tar.bz2
freezegun                 0.2.2                    py27_1    http://conda:82/ci/win-64/freezegun-0.2.2-py27_1.tar.bz2
funcdesigner              0.5201                    <pip>
funcsigs                  0.4                      py27_0    <unknown>
futures                   3.0.3                    py27_0    http://conda:82/ci/win-64/futures-3.0.3-py27_0.tar.bz2
hdf5                      1.8.15.1                      2    <unknown>
html5lib                  0.999                    py27_0    http://conda:82/ci/win-64/html5lib-0.999-py27_0.tar.bz2
ipython                   3.2.1                    py27_0    http://conda:82/ci/win-64/ipython-3.2.1-py27_0.tar.bz2
jinja2                    2.7.3.post98.g85820fc          py27_0    http://conda:82/ci/win-64/jinja2-2.7.3.post98.g85820fc-py27_0.tar.bz2
jsonschema                2.4.0                    py27_0    <unknown>
libpython                 1.0                      py27_1    http://conda:82/ci/win-64/libpython-1.0-py27_1.tar.bz2
line-profiler             1.0                       <pip>
line_profiler             1.0.post5.ga2f484a          py27_0    http://conda:82/ci/win-64/line_profiler-1.0.post5.ga2f484a-py27_0.tar.bz2
lxml                      3.4.4                    py27_0    <unknown>
markupsafe                0.23                     py27_0    <unknown>
matplotlib                1.4.3                np19py27_1    <unknown>
mechanize                 0.2.5                    py27_2    http://conda:82/ci/win-64/mechanize-0.2.5-py27_2.tar.bz2
memory-profiler           0.33                      <pip>
memory_profiler           0.33                     py27_0    http://conda:82/ci/win-64/memory_profiler-0.33-py27_0.tar.bz2
mistune                   0.7                      py27_0    http://conda:82/ci/win-64/mistune-0.7-py27_0.tar.bz2
mkl                       11.3.1                        0    defaults
mock                      1.2.0                    py27_0    http://conda:82/ci/win-64/mock-1.2.0-py27_0.tar.bz2
mpi4py                    1.3.1.post152.g2b55fb2          py27_0    http://conda:82/ci/win-64/mpi4py-1.3.1.post152.g2b55fb2-py27_0.tar.bz2
mpltools                  0.1.post21.g1d4c339          py27_1    http://conda:82/ci/win-64/mpltools-0.1.post21.g1d4c339-py27_1.tar.bz2
mpmath                    0.19.post3.g5cb118a      np19py27_1    http://conda:82/ci/win-64/mpmath-0.19.post3.g5cb118a-np19py27_1.tar.bz2
msvc_runtime              1.0.1                     vc9_0  [vc9]  defaults
multipledispatch          0.4.8                    py27_0    http://conda:82/ci/win-64/multipledispatch-0.4.8-py27_0.tar.bz2
mxbase                    3.2.7                    py27_1    http://conda:82/ci/win-64/mxbase-3.2.7-py27_1.tar.bz2
networkx                  1.9.1                    py27_0    <unknown>
nlopt                     2.2.4                np19py27_1    http://conda:82/ci/win-64/nlopt-2.2.4-np19py27_1.tar.bz2
nose                      1.3.7                    py27_0    <unknown>
numdifftools              0.6.0                np19py27_1    http://conda:82/ci/win-64/numdifftools-0.6.0-np19py27_1.tar.bz2
numexpr                   2.4.3                np19py27_0    <unknown>
numpy                     1.9.2                    py27_0    <unknown>
numpydoc                  0.5.post1.g7c3a1dc          py27_0    http://conda:82/ci/win-64/numpydoc-0.5.post1.g7c3a1dc-py27_0.tar.bz2
openopt                   0.5.2                np19py27_4    http://conda:82/ci/win-64/openopt-0.5.2-np19py27_4.tar.bz2
packaging                 15.3                     py27_0    http://conda:82/ci/win-64/packaging-15.3-py27_0.tar.bz2
pandas                    0.15.2               np19py27_1    http://conda:82/ci/win-64/pandas-0.15.2-np19py27_1.tar.bz2
patsy                     0.3.0                np19py27_0    <unknown>
pbr                       1.3.0                    py27_0    http://conda:82/ci/win-64/pbr-1.3.0-py27_0.tar.bz2
pip                       8.0.2                    py27_0    defaults
protobuf                  3.0.0a3.post418+g0cb84ee          py27_0    http://conda:82/ci/win-64/protobuf-3.0.0a3.post418+g0cb84ee-py27_0.tar.bz2
psutil                    2.2.1                    py27_0    <unknown>
pyamg                     2.2.1                np19py27_0    http://conda:82/ci/win-64/pyamg-2.2.1-np19py27_0.tar.bz2
pydevd                    4.0.5                    py27_0    http://conda:82/ci/win-64/pydevd-4.0.5-py27_0.tar.bz2
pygments                  2.0.2                    py27_0    <unknown>
pymkl                     0.1.0.post051.gff66387          py27_0    http://conda:82/ci/win-64/pymkl-0.1.0.post051.gff66387-py27_0.tar.bz2
pyodbc                    3.0.10                   py27_0    http://conda:82/ci/win-64/pyodbc-3.0.10-py27_0.tar.bz2
pyparsing                 2.0.3                    py27_0    <unknown>
pyqt                      4.10.4                   py27_1    <unknown>
pyreadline                2.0                      py27_0    <unknown>
pytables                  3.2.0                np19py27_0    <unknown>
python                    2.7.11                        2    defaults
python-dateutil           2.4.1                    py27_0    http://conda:82/ci/win-64/python-dateutil-2.4.1-py27_0.tar.bz2
python-ntlm               1.1.0                    py27_0    http://conda:82/ci/win-64/python-ntlm-1.1.0-py27_0.tar.bz2
pythondotnet              2.0.0.2.post1.b46375e2          py27_0    http://conda:82/ci/win-64/pythondotnet-2.0.0.2.post1.b46375e2-py27_0.tar.bz2
pytz                      2015.4                   py27_0    <unknown>
pywin32                   219                      py27_0    <unknown>
pyzmq                     14.7.0                   py27_0    <unknown>
requests                  2.7.0                    py27_0    <unknown>
requests-ntlm             0.0.2.2.post25.ge76408a          py27_1    http://conda:82/ci/win-64/requests-ntlm-0.0.2.2.post25.ge76408a-py27_1.tar.bz2
sarge                     0.1.3                    py27_1    http://conda:82/ci/win-64/sarge-0.1.3-py27_1.tar.bz2
scikit-learn              0.16.1               np19py27_0    <unknown>
scipy                     0.15.1               np19py27_0    <unknown>
scs                       1.0.2.post93.g8c3f9d7      np19py27_0    http://conda:82/ci/win-64/scs-1.0.2.post93.g8c3f9d7-np19py27_0.tar.bz2
setuptools                19.6.2                   py27_0    defaults
six                       1.9.0                    py27_0    <unknown>
snowballstemmer           1.2.0                    py27_0    <unknown>
sortedcontainers          0.9.4                    py27_0    http://conda:82/ci/win-64/sortedcontainers-0.9.4-py27_0.tar.bz2
spacefuncs                0.52                      <pip>
sphinx                    1.3.1                    py27_0    <unknown>
sphinx-rtd-theme          0.1.7                     <pip>
sphinx_rtd_theme          0.1.7                    py27_0    <unknown>
sqlalchemy                1.0.7                    py27_0    http://conda:82/ci/win-64/sqlalchemy-1.0.7-py27_0.tar.bz2
ssl_match_hostname        3.4.0.2                  py27_0    <unknown>
statsmodels               0.6.1                np19py27_0    <unknown>
sympy                     0.7.6                    py27_0    <unknown>
tables                    3.2.0                     <pip>
toolz                     0.7.2                    py27_0    <unknown>
tornado                   4.1                      py27_0    http://conda:82/ci/win-64/tornado-4.1-py27_0.tar.bz2
tqdm                      0.1.pre14.gda5fdbf7c0          py27_0    http://conda:82/ci/win-64/tqdm-0.1.pre14.gda5fdbf7c0-py27_0.tar.bz2
wheel                     0.26.0                   py27_1    defaults
xlrd                      0.9.4                    py27_0    http://conda:82/ci/win-64/xlrd-0.9.4-py27_0.tar.bz2
xlsxwriter                0.7.3                    py27_0    <unknown>
zlib                      1.2.8                         0    <unknown>
```

**Comment by kalefranz** [comment]
Cloning has been fixed in 4.3, if not 4.2.
